lib/src/interfaces/NavigationComponentProps.ts

```diff
@@ -1,3 +1,4 @@
 export interface NavigationComponentProps {
   componentId: string;
+  componentName: string;
```

Comment:
Shouldn't this be `componentName?: string`, to avoid breakage of existing code?

Reply:
I will revert this for now, It doesn't make a lot of sense to add it as optional as it always exist